Mlem/Views/Tabs/Profile/UserView.swift

```diff
@@ -0,0 +1,282 @@
+                                        Text("Comments (\(abbreviateNumber(user.commentCount ?? 0)))")
+                                    case .communities:
+                                        Text("Communities (\(abbreviateNumber(user.moderatedCommunities?.count ?? 0)))")
+                                    default:
+                                        Text(tab.label)
+                                    }
+                                }
+                                .padding(.horizontal, AppConstants.postAndCommentSpacing)
+                                .padding(.vertical, 4)
+                            }
+                            .scrollIndicators(.hidden)
+                            Divider()
+                            UserFeedView(
+                                user: user,
+                                privatePostTracker: privatePostTracker,
+                                privateCommentTracker: privateCommentTracker,
+                                communityTracker: communityTracker,
+                                selectedTab: $selectedTab
+                            )
+                        }
+                        .transition(.opacity)
+                    }
+                }
+                .animation(.easeOut(duration: 0.2), value: isLoadingContent)
+            }
+        }
+        .environmentObject(privatePostTracker)
+        .environmentObject(privateCommentTracker)
+        .destructiveConfirmation(
+            isPresentingConfirmDestructive: $isPresentingConfirmDestructive,
+            confirmationMenuFunction: confirmationMenuFunction
+        )
+        .toolbar {
+            ToolbarItem(placement: .topBarTrailing) {
+                let functions = user.menuFunctions { user = $0 }
+                if functions.count == 1, let first = functions.first {
+                    MenuButton(menuFunction: first, confirmDestructive: confirmDestructive)
+                } else {
+                    Menu {
+                        ForEach(functions) { item in
+                            MenuButton(menuFunction: item, confirmDestructive: confirmDestructive)
+                        }
+                    } label: {
+                        Label("Menu", systemImage: Icons.menu)
+                    }
+                }
+            }
+            if isOwnProfile {
+                ToolbarItem(placement: .topBarLeading) {
+                    Button("Switch Account", systemImage: Icons.switchUser) {
+                        isPresentingAccountSwitcher = true
+                    }
+                }
+            }
+        }
+        .task(priority: .userInitiated) {
+            if isLoadingContent {
+                await tryReloadUser()
+            }
+        }
+        .onChange(of: user.userId) { _ in
+            Task {
+                await tryReloadUser()
+            }
+        }
+        .refreshable {
+            await tryReloadUser()
```

Comment:
This call gets cancelled by the view redraw--either state changes within the function need to be delayed until after all calls complete or the whole thing can just be wrapped in a task (value awaited so that the `refreshable` indicator waits properly)

```suggestion
            await Task {
                await tryReloadUser()
            }.value
```